src/generators/default/string/string.test.ts

```diff
@@ -16,6 +17,7 @@ import { OptionalGenerator } from '../optional';
 describe('create strings', () => {
 	const core = new Core().register([
 		UuidGenerator,
+		CuidGenerator,
```

Comment:
I originally had a cuid generator, but saw that cuid has been deprecated for security vulnerabilities. In an effort to encourage people to migrate to cuid2, I purposefully left it out. Maybe this isn't the right platform to do that and having parity with zod is our priority. I partially feel responsible for enabling people to do bad security practices by providing tools to use insecure products 🤷🏼.  I'll let you make the call, though.

Reply:
I was looking at the test coverage and I noticed the Cuid generator had zero coverage.
I started with implementing it and creating a test for it, but then I reverted those changes for the reasons you mentioned.

The CuidGenerator is added here to add a test for it to see if it throws.